src/mgmt/srv_target.c

```diff
@@ -275,6 +275,11 @@ cleanup_leftover_cb(uuid_t uuid, void *arg)
 
 	/* destroy blobIDs */
 	D_DEBUG(DB_MGMT, "Clear SPDK blobs for pool "DF_UUID"\n", DP_UUID(uuid));
+	rc = vos_pool_kill(uuid, VOS_POF_RDB);
```

Comment:
This vos_pool_kill() is called in pool_hop_free() as well, but I think we missed the flag on that call. I think the vos_pool_kill(pool->vp_id, 0) in pool_hop_free() should be changed to vos_pool_kill(pool->vp_id, pool->vp_rdb ? VOS_POF_RDB : 0);

Reply:
@NiuYawei, indeed, I missed `pool_hop_free`. Changed in the way suggested.